### DISCUSSION TITLE
Languages for repo with MD extension is not showing up

### DISCUSSION THREAD

**Discussion by ahmadalwareh** [github-linguist/linguist#6238] [Classification] [answered]
## Describe the bug

I have this repo: https://github.com/ahmadalwareh/Python_Interview_Questions
and `Languages` is not detected or viewed

### Expected behavior
Repo get `Markdown` tag

**Comment by lildude** [comment, marked as answer]
This is expected documented behaviour. From the ["How Linguist Works" documentation](https://github.com/github/linguist/blob/master/docs/how-linguist-works.md#how-linguist-works):

> Linguist starts by going through all the files in a repository and excludes all files that it determines to be binary data, [vendored code](https://github.com/github/linguist/blob/master/docs/overrides.md#vendored-code), [generated code](https://github.com/github/linguist/blob/master/docs/overrides.md#generated-code), [documentation](https://github.com/github/linguist/blob/master/docs/overrides.md#documentation), or are defined as `data` (e.g. SQL) or `prose` (e.g. Markdown) languages, whilst taking into account any [overrides](https://github.com/github/linguist/blob/master/docs/overrides.md).

Markdown is prose so you'll need to implement the `linguist-detectable` override if you want Markdown to show in the languages list.

**Reply by ahmadalwareh** [reply]
I've updated the `.gitattributes` with a `linguist-detectable` option, how long does it take to see the changes?

**Reply by lildude** [reply]
You'll need to push a change to one of the `.md` files in order to force a re-analysis which will consider them.

**Reply by ahmadalwareh** [reply]
Thanks for helping @lildude !
Just to make sure; Do the updates into `.gitattributes` added correctly?

**Reply by lildude** [reply]
Yup, but I've just remembered, these are also considered documentation so you need to override that too using the `-linguist-documentation` attribute.

This single rule should do the trick for your repo:

```gitattributes
*.md linguist-detectable -linguist-documentation
```

**Reply by ahmadalwareh** [reply]
Thanks man!
